jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -908,11 +909,48 @@ public void testGetCreationTime_useCurrentTimestamp() throws InvalidCreationTime
 
   @Test
   public void testGetCreationTime_isoDateTimeValue() throws InvalidCreationTimeException {
-    Instant time =
-        PluginConfigurationProcessor.getCreationTime(
-            "2011-12-03T10:15:30+09:00", projectProperties);
     Instant expected = DateTimeFormatter.ISO_DATE_TIME.parse("2011-12-03T01:15:30Z", Instant::from);
-    Assert.assertEquals(expected, time);
+    List<String> validTimeStamps =
+        ImmutableList.of(
+            "2011-12-03T10:15:30+09:00",
+            "2011-12-03T10:15:30+09:00[Asia/Tokyo]",
+            "2011-12-02T16:15:30-09:00",
+            "2011-12-03T10:15:30+0900",
+            "2011-12-02T16:15:30-0900",
+            "2011-12-03T10:15:30+09",
+            "2011-12-02T16:15:30-09",
+            "2011-12-03T01:15:30Z");
+    for (String timeString : validTimeStamps) {
+      Instant time = PluginConfigurationProcessor.getCreationTime(timeString, projectProperties);
+      Assert.assertEquals("for " + timeString, expected, time);
+    }
+  }
+
+  @Test
+  public void testGetCreationTime_isoDateTimeValueTimeZoneRegionOnlyAllowedForMostStrict8601Mode() {
+    List<String> invalidTimeStamps =
+        ImmutableList.of(
+            "2011-12-03T01:15:30+0900[Asia/Tokyo]", "2011-12-03T01:15:30+09[Asia/Tokyo]");
+    for (String timeString : invalidTimeStamps) {
+      try {
+        PluginConfigurationProcessor.getCreationTime(timeString, projectProperties);
+        Assert.fail(
+            "creationTime should fail if region specified when zone not in HH:MM mode - "
```

Comment:
Hmm... is this true with the built-in `ISO_DATE_TIME`?

Reply:
So ISO_DATE_TIME requires you use `HH:MM`, I don't think we have a reason to add in the code to parse the regionId, and this test just verifies that behavior